code/src/OSFramework/Feature/IConditionalFormat.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.Feature {
+    export interface IConditionalFormat {
+        addRule(
+            binding: string,
+            rule: Array<OSFramework.OSStructure.ConditionalFormat>
+        ): void;
```

Comment:
I think that we are missing a removeRule.

Reply:
done